content/en/guides/core/registry/model_registry_eol.md

```diff
@@ -4,105 +4,153 @@ menu:
     identifier: model_registry_eol
     parent: registry
 title: Migrate from legacy Model Registry
-weight: 8
+weight: 9
 ---
 
-W&B will transition assets from the legacy [W&B Model Registry]({{< relref "/guides/core/registry/model_registry/" >}}) to the new [W&B Registry]({{< relref "./" >}}). This migration will be fully managed and triggered by W&B, requiring no intervention from users. The process is designed to be as seamless as possible, with minimal disruption to existing workflows.
+W&B is migrating from the legacy **Model Registry** to the enhanced **W&B Registry**. This transition is designed to be seamless, fully managed by W&B, and will preserve your workflows while unlocking powerful new features. For any questions or support, contact [support@wandb.com](mailto:support@wandb.com).
```

Comment:
The second sentence could use a bit of massaging. "designed to be" doesn't flow well into "fully managed by W&B" or "will preserve your workflows". Maybe worth splitting that sentence up.